lib/routes/drivers/tests/routes.test.js

```diff
@@ -0,0 +1,167 @@
+                new Promise((resolve, reject) => {
+                    reject(new Error('Test Error'));
+                });
+        });
+
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers',
+            payload
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Test Error');
+    });
+
+    it('should return 200', async () => {
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload
+        });
+        expect(res.statusCode).toEqual(200);
+    });
+
+    it('should return 400', async () => {
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload: {}
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Invalid request payload input');
+    });
+
+    it('should return 400', async () => {
+        server = await resetAndMockDB(async allDbs => {
+            allDbs.drivers.update = () =>
+                new Promise((resolve, reject) => {
+                    reject(new Error('Test Error'));
+                });
+        });
+
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Test Error');
+    });
+
+    it('should return 200', async () => {
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers/add/location',
+            payload: payloadForLocation
+        });
+        expect(res.statusCode).toEqual(200);
+    });
+
+    it('should return 400', async () => {
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers/add/location',
+            payload: {}
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Invalid request payload input');
+    });
+
+    it('should return 400', async () => {
+        server = await resetAndMockDB(async allDbs => {
+            allDbs.driver_location.create = () =>
+                new Promise((resolve, reject) => {
+                    reject(new Error('Test Error'));
+                });
+        });
+
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers/add/location',
+            payload: payloadForLocation
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Test Error');
+    });
+
+    it('should return 200', async () => {
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers/accept',
+            payload: payloadForAccepting
+        });
+        expect(res.statusCode).toEqual(200);
+    });
+
+    it('should return 400', async () => {
```

Comment:
it should return 400 when <this is passed>,
lets have a little details within tests

Reply:
Done.